version.properties

```diff
@@ -1,2 +1,2 @@
-version=2.2.0-beta.2
+version=2.2.0
```

Comment:
Things in JCenter shouldn't be rather removed as well (it's the primary artifacts repo for Gradle projects by default).